src/platform/graphics/graphics-device.js

```diff
@@ -40,11 +40,11 @@ class GraphicsDevice extends EventHandler {
     isWebGPU = false;
 
     /**
-     * The type of graphics device that the system could initialize. Can be one of pc.DEVICETYPE_WEBGL1, pc.DEVICETYPE_WEBGL2 or pc.DEVICETYPE_WEBGPU.
+     * The type of the device. Can be one of pc.DEVICETYPE_WEBGL1, pc.DEVICETYPE_WEBGL2 or pc.DEVICETYPE_WEBGPU.
```

Comment:
Sorry, I should have realised this earlier. Can we rename this to _deviceType, and make deviceType a getter only (read only access), as currently this feels like a property users can change.

Reply:
Yep that makes sense.